client/i18n/locales/english/translations.json

```
@@ -444,8 +444,8 @@
     "min-50-max-500": "50 character minimum, 500 character maximum",
     "rsa": "Read, search, ask",
     "rsa-forum": "<strong>Before making a new post</strong> please <0>check if your question has already been answered on the forum</0>.",
-    "reset": "Reset this lesson?",
-    "reset-warn": "Are you sure you wish to reset this lesson? The editors and tests will be reset.",
+    "reset": "Reset this lesson ({{ title }})?",
+    "reset-warn": "Are you sure you wish to reset {{ title }}? The code editors and tests will be reset to how they were last step.",
```

```suggestion
    "reset-warn": "Are you sure you wish to reset {{ title }}? The code editors and tests will be reset.",
```

the change would not work for labs